number-filtering.js

```diff
@@ -0,0 +1,66 @@
+let myNumbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
+
+let numbers = myNumbers;
+
+function filterEven(){
+    for(let num of numbers){
+      if(num % 2 == 0){
+        console.log(num)
+      }
+    }
+}
+
+filterEven();
+
+// task 4
+
+const userScore = [  
+    {id: 1, name: 'Olarike', score: 34},  
+    {id: 2, name: 'Taiwo', score: 12},  
+    {id: 3, name: 'Kehinde', score: 27} ];
+
+// Find where user score above 12 and return the user info.
+
+function returnUser(){
+    let user = userScore.filter(item => item.score > 12);
+    console.log(user)
+}
+
+returnUser();
+
+
+function findScore(){
+    Score = userScore.filter(item => item.score)
+    Score.sort((function(a, b){return b.score - a.score}))
+    console.log(Score)
+}
+
+findScore()

```

Comment:
the method naming is mis-leading, kindly use a naming that represent what you're trying to do.

I believe you wanted to sort the array.

Also there is no need of filtering.